proto/grp.proto

```diff
@@ -61,6 +64,41 @@ message GetGroupResponse {
   string owned_domain = 3;
 }
 
+message GetGroupUsersRequest {
```

Comment:
Going back to we talked about before -- let's take an explicit group ID here in the request and leave the request_context group ID for the selector only.

Reply:
Done